prow/config.yaml

```diff
@@ -473,6 +473,10 @@ tide:
 push_gateway:
   endpoint: pushgateway
 
+github_reporter: 
+  job_types_to_report:
+  - presubmit
```

Comment:
how do we prevent double reporting w/ plank? ... catching up on this work ...

Reply:
er so we're not reporting presubmits by default?? where can I read about this change?

Reply:
we are

Reply:
see https://github.com/kubernetes/test-infra/blob/master/prow/config/config.go#L774-L785

Reply:
Ok thanks.

Reply:
FYI would help if that error told you want were valid values for the types, instead of making someone read the code

Reply:
Also do we have ANNOUNCEMENTS.md for this?

Reply:
> FYI would help if that error told you want were valid values for the types, instead of making someone read the code

👍 

> Also do we have ANNOUNCEMENTS.md for this?

yes - https://github.com/kubernetes/test-infra/blob/master/prow/ANNOUNCEMENTS.md - first item in new features

Reply:
As an admin trying to figure out what I should be doing to keep up to date with the latest changes that is not super useful -- it's a good overview of the config feature but doesn't tell me how to migrate.

Reply:
What steps do I need to take?

Reply:
> What steps do I need to take?

nothing if you want to keep the existing behavior (only report for presubmit jobs)

add that trunk of config to config.yaml if you want to report on postsubmit jobs